system/include/emscripten/webaudio.h

```diff
@@ -57,7 +58,7 @@ void emscripten_destroy_audio_context(EMSCRIPTEN_WEBAUDIO_T audioContext);
 // Disconnects the given audio node from its audio graph, and then releases
 // the JS object table reference to the given audio node. The specified handle
 // is invalid after calling this function.
-void emscripten_destroy_web_audio_node(EMSCRIPTEN_WEBAUDIO_T objectHandle);
+void emscripten_destroy_web_audio_node(EMSCRIPTEN_AUDIO_WORKLET_NODE_T objectHandle);
```

Comment:
Can this API not be used without audio worklets?

Reply:
These types end up being interchangeable sometimes, see here for example:

https://github.com/emscripten-core/emscripten/blob/d19c16213cdc6f63400bf1a55b3c76704ba663d5/src/library_webaudio.js#L324

Reply:
> Can this API not be used without audio worklets?

Parts of it, but the it needs JS to create other node types.

(In general though, some of the functions need a node, others a worklet context, and often they’re interchangeable when creating a graph depending on the use.)

Reply:
If the types should be interchangeable, then perhaps `EMSCRIPTEN_AUDIO_WORKLET_NODE_T` can just be removed entirely, and `EMSCRIPTEN_WEBAUDIO_T` used everywhere?  I can update to do that if preferred.

Reply:
Some parts of the API only work with a single type, others have tests like this and work with node or context:

https://github.com/emscripten-core/emscripten/blob/d19c16213cdc6f63400bf1a55b3c76704ba663d5/src/library_webaudio.js#L327

I don't think there's a correct choice.